ngrok-java-native/src/lib.rs

```diff
@@ -392,12 +392,18 @@ impl<'local> com_ngrok::NativeSessionRs<'local> for NativeSessionRsImpl<'local>
             bldr = bldr.handle_heartbeat(HeartbeatCallback::from(self.env, heartbeat_obj));
         }
 
-        let user_agents = jsb.get_user_agents(self.env);
-        for i in 0..user_agents.size(self.env) {
-            let user_agent: ComNgrokSessionUserAgent = user_agents.get(self.env, i).into();
-            bldr = bldr.child_client(
-                user_agent.get_name(self.env),
-                user_agent.get_version(self.env),
+        let client_infos = jsb.get_client_infos(self.env);
+        for i in 0..client_infos.size(self.env) {
+            let client_info: ComNgrokSessionClientInfo = client_infos.get(self.env, i).into();
+            let comments = if client_info.has_comments(self.env) {
+                Option::<String>::Some(client_info.get_comments(self.env))
+            } else {
+                Option::<String>::None
+            };
+            bldr = bldr.client_info(
+                client_info.get_type(self.env),
+                client_info.get_version(self.env),
+                comments,
```

Comment:
Not sure if the order matters, but `versions` are now [pushed like a stack](https://github.com/ngrok/ngrok-rust/blob/21196faa79d9802756037c5dabb27ece7e80ee89/ngrok/src/session.rs#L643), i.e. the last one added goes to the front.

Reply:
🤔 I'll take a look again, but I think what I see as a final result it still correct, e.g.:
<img width="590" alt="Screenshot 2023-09-14 at 11 29 19 AM" src="https://github.com/ngrok/ngrok-java/assets/45884158/4785c942-6cec-4bfd-b89b-132543fe1823">

Reply:
Yeah, lgtm :+1:

Doesn't matter that they aren't getting pushed stackfully in java code, as long as the order is order is consistent when translating to rust.

Looks like the info is getting appended to an array, and then we iterate forwards over that array, which keeps the order the same.